include/swift/AST/Attr.def

```diff
@@ -713,6 +713,12 @@ DECL_ATTR(exclusivity, Exclusivity,
   ABIStableToAdd | ABIStableToRemove | APIStableToAdd | APIStableToRemove,
   128)
 
+DECL_ATTR(_backDeploy, BackDeploy,
+  OnAbstractFunction |
+  AllowMultipleAttributes | LongAttribute | UserInaccessible |
```

Comment:
Do we want `AllowMultipleAttributes` or is one attribute listing many platforms enough?

Reply:
I chose to allow multiple attributes because currently this attribute explodes into one attribute per platform and version tuple when printed into the interface (following `@_originallyDefinedIn` as an archetype).

Reply:
Ah, sounds good.